vagrant/provision_experiment_environment.sh

```diff
@@ -141,6 +141,14 @@ wget --no-verbose https://github.com/intelsdi-x/snap-plugin-publisher-file/relea
 wget --no-verbose https://github.com/intelsdi-x/snap-plugin-collector-use/releases/download/${SNAP_PLUGIN_COLLECTOR_USE_VERSION}/snap-plugin-collector-use_linux_x86_64 -O ${SWAN_BIN}/snap-plugin-collector-use
 
 
+echo "----------------------------- Pulling docker image (`date`)"
+docker pull intelsdi/swan
```

Comment:
Both of 'docker pull' and 'retrieve binaries' are already in line 103. Please remove.